### DISCUSSION TITLE
Workflow Update Content help

### DISCUSSION THREAD

**Discussion by draikolz** [OrchardCMS/OrchardCore#9014] [Q&A]
I am wondering if anyone can point me to a good way to update my contents via workflow.
During update should we just pass the updated portion of JSON or the whole JSON?

Tq

**Comment by hishamco** [comment]
There's a `UpdateContentTask`

**Reply by draikolz** [reply]
Its really slow and I dont seem to be able to update my contents.
Can anyone show me a sample that I can base on?

**Reply by ns8482e** [reply]
I believe, You can pass the updated portion and it will merge the updated data with original data.
